VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -442,7 +442,7 @@ def get_total_used_size(self):
                 time.sleep(1)
```

Comment:
Suggestion: I know that this is the existing code. Can we refactor by creating a new method that takes command as input and output as a response?

I see that this being used at other places. this helps in future changes to update in a single place.

Reply:
done